src/components/ProgressBar/index.scss

```diff
@@ -0,0 +1,36 @@
+@use './../../styles/base.scss' as *;
+
+.progress-bar {
+  width: 100%;
+
+  margin: 0;
+  padding-top: 20px;
+  padding-right: 40px;
+  padding-left: 40px;
+}
+
+.progress-bar-background {
+  width: 100%;
+  height: 3px;
+
+  margin: 0;
+  padding: 0;
+
+  position: relative;
+
+  background-color: $grey-150;
+  border-radius: 3px;
+}
+
+.progress-bar-foreground {
+  height: 6px;
+
+  position: absolute;
+  top: -50%;
+  left: 0;
+
+  background-color: $yellow-500;
+  border-radius: 3px;
+
+  transition: width 1s ease-in-out;
+}
```

Comment:
Estamos usando sass e BEM dá pra usar o nesting do sass https://sass-lang.com/blog/sass-and-native-nesting/